src/secureCreatives.js

```diff
@@ -74,21 +98,43 @@ function sendAdToCreative(adObject, remoteDomain, source) {
 function resizeRemoteCreative({ adUnitCode, width, height }) {
   // resize both container div + iframe
   ['div', 'iframe'].forEach(elmType => {
-    let elementStyle = getElementByAdUnit(elmType).style;
-    elementStyle.width = `${width}px`;
-    elementStyle.height = `${height}px`;
+    const element = getElementByAdUnit(elmType);
+    if (typeof element !== 'undefined') {
+      if (element === null) {
+        events.emit(ERROR_SECURE_CREATIVE, {
+          msg: 'No element of type "elmType" for adUnit',
+          adUnitCode,
+          elmType,
+          width,
+          height
+        });
+      } else {
+        let elementStyle = element.style;
+        elementStyle.width = `${width}px`;
+        elementStyle.height = `${height}px`;
+      }
+    }
   });
   function getElementByAdUnit(elmType) {
-    return document
-      .getElementById(
-        find(
-          window.googletag
-            .pubads()
-            .getSlots()
-            .filter(isSlotMatchingAdUnitCode(adUnitCode)),
-          slot => slot
-        ).getSlotElementId()
-      )
-      .querySelector(elmType);
+    const element = document.getElementById(
+      find(
```

Comment:
Can't find any documentation about this ```find```. It seems to exist on chrome though.

Reply:
@alexarvanitidismrf It's imported from the `core-js` lib: `import find from 'core-js/library/fn/array/find';`

Reply:
From what I see, this gets transpiled as a `Function.apply`, which makes it fine!